nashvillefcc/nashvillefcc2020website
# Meetup api for events#24

### Description of Changes

- Added a Netlify function **pingEvents.js** which contacts RSStoJSON api and gets the current Meetup events information.
- Netlify site integrated with FaunaDB which stores (caches) the current events data for 8 hours.
- Added logic to index.js to process the data received from pingEvents function and distribute it to individual event cards.
- Changed the style in eventCards so that the thumbnail is displayed at its full width.

### How can I test this?

1. Clone the files from github onto your local workspace.
2. Run `npm install` to install needed dependencies.
3. Once finished, test to make sure the Netlify CLI is installed by typing `netlify` in your console. If not type `npm install netlify-cli -g` in the console to install it.
4. Once confirmed, in index.js uncomment  `'http://localhost:8888/.netlify/functions/pingEvents'` to access the Netlify function.
5. Then type in `netlify dev` to start the development server.
6. Then you are ready to test!
